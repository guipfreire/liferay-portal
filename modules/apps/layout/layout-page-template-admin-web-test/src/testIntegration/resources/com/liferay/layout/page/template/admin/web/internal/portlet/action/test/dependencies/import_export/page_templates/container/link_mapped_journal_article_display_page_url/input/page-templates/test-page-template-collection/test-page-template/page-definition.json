{
	"pageElement": {
		"pageElements": [
			{
				"definition": {
					"fragmentLink": {
						"href": {
							"defaultFragmentInlineValue": {
								"value": "${DISPLAY_PAGE_URL}"
							},
							"mapping": {
								"fieldKey": "displayPageURL",
								"itemReference": {
									"className": "com.liferay.journal.model.JournalArticle",
									"classPK": "${CLASS_PK}"
								}
							}
						}
					}
				},
				"type": "Section"
			}
		],
		"type": "Root"
	},
	"settings": {
		"colorSchemeName": "01",
		"themeName": "Classic"
	}
}