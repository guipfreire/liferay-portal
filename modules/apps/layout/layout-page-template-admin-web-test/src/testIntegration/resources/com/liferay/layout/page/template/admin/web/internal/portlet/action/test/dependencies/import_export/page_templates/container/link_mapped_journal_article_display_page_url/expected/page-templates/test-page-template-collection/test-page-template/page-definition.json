{
	"pageElement": {
		"pageElements": [
			{
				"definition": {
					"backgroundFragmentImage": {
					},
					"fragmentLink": {
						"href": {
							"defaultFragmentInlineValue": {
								"value": "${DISPLAY_PAGE_URL}"
							},
							"mapping": {
								"fieldKey": "displayPageURL",
								"itemReference": {
									"className": "com.liferay.journal.model.JournalArticle",
									"classPK": "${CLASS_PK}"
								}
							}
						}
					},
					"layout": {
						"borderWidth": 0,
						"contentDisplay": "Block",
						"marginBottom": 0,
						"marginLeft": 0,
						"marginRight": 0,
						"marginTop": 0,
						"opacity": 100,
						"paddingBottom": 0,
						"paddingLeft": 0,
						"paddingRight": 0,
						"paddingTop": 0,
						"widthType": "Fluid"
					}
				},
				"type": "Section"
			}
		],
		"type": "Root"
	},
	"settings": {
		"colorSchemeName": "01",
		"themeName": "Classic"
	}
}